util/system/path.go

```diff
@@ -71,17 +70,6 @@ func NormalizePath(parent, newPath, inputOS string, keepSlash bool) (string, err
 		newPath = path.Join(parent, newPath)
 	}
 
-	if keepSlash {
```

Comment:
@tonistiigi -- you were right in your observation earlier:
>  but ideally we should keep the functionality to keep slash into a single function and not multiple, and NormalizePath should not be linux-specific.

So, this was redudant after handling `keepSlash` in `CheckSystemDriveAndRemoveDriveLetter`. I think the original intent of this was to retain the slash being stripped off in `CheckSystemDriveAndRemoveDriveLetter` -> `filepath.Clean()`.

/cc. @gabriel-samfira

Reply:
will revisit this in further optimization/refactoring, must have been wrong on having it on one place; for now.